### DISCUSSION TITLE
Outside Areas seen from Barrows

### DISCUSSION THREAD

**Discussion by Baconade** [runelite/runelite#17355] [Ideas]
![Barrows 1](https://github.com/runelite/runelite/assets/156228508/523de8b2-b70a-4c6d-a1b1-15a330c29b63)
![Barrows 2](https://github.com/runelite/runelite/assets/156228508/d6e2fe93-2521-4364-acca-f5b520080ae6)

Here are two outside areas that can be seen from inside the barrows crypts and brother mounds

**Comment by Nightfirecat** [comment]
I've created an issue from this discussion for better tracking: #17356
